turbinia/processors/mount_local.py

```diff
@@ -208,7 +208,7 @@ def GetFilesystem(path):
   Returns:
     str: the filesystem detected (for example: 'ext4')
   """
-  cmd = ['lsblk', path, '-f', '-o', 'FSTYPE', '-n']
+  cmd = ['fsstat', '-t', path]
```

Comment:
add a small Extra: line to the PR description so we know we replaced lsblk with fsstat.